docs/mkdocs.yml

```diff
@@ -96,6 +96,7 @@ plugins:
             group_by_category: true
             show_category_heading: false
             show_root_heading: true
+            inherited_members: true
```

Comment:
Did it worked?

Reply:
It did :) the option literally got released last week in https://github.com/mkdocstrings/python/releases/tag/1.2.0

Reply:
that's what it's called being lucky :)